### PR TITLE
expose option for git submodule update

### PR DESCRIPTION
**Pull request by bertsky**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,6 +14,7 @@ PIP_OPTIONS_E = $(filter-out -e,$(PIP_OPTIONS))
 # Set to 1 to skip all submodule updates. For development.
 NO_UPDATE = 0
 GIT_RECURSIVE = # --recursive
+GIT_DEPTH = # --depth 1
```
Comment (stweil) [inline Makefile:17]: Technically the changes look fine. Personally I'd prefer less macros. What about combining `GIT_RECURSIVE` and `GIT_DEPTH` into a single new `GIT_SUBMODULE_UPDATE_OPTIONS` (or just `GIT_OPTIONS`)?

Comment (stweil) [reply Makefile:17]: Ah, sorry, that would not work because `GIT_RECURSIVE` is also used in other git operations.

Comment (bertsky) [reply Makefile:17]: Yes, exactly. Personally I think this PR is just another try to at least get _something_ running again (fixing both network I/O failures and extremely long build times recently).

**Review by kba — APPROVED** [review]
Valid point about too many macros but the ends justify the means here imho.

**Comment by bertsky** [comment]
CI for the PR branch is unaffected of course. @kba shall we give it a try?

**Comment by bertsky** [comment]
```
#!/bin/bash -eo pipefail
docker push ocrd/all:minimum &
docker push ocrd/all:minimum-git &
docker push ocrd/all:medium &
docker push ocrd/all:medium-git &
docker push ocrd/all:maximum &
docker push ocrd/all:maximum-git &
docker push ocrd/all:maximum-cuda &
docker push ocrd/all:maximum-cuda-git &
wait -f
/bin/bash: line 8: wait: -f: invalid option
wait: usage: wait [-n] [id ...]
```

Damnit. `wait -f` was only introduced in bash 5.0. I'll make another PR to fix this. (Unfortunately, the new Docker images do not seem to have arrived at DH. I reckon the bg jobs were all killed when `wait` failed.)

**Comment by kba** [comment]
> > I'll make another PR to fix this.
> 
> @kba is it okay if I commit the CircleCI fix directly to master this time?

Go ahead. Pull first though because I misread  the output and thought the build had simply timed out and increased `no_output_timeout` and pushed to master, sorry about that.

**Comment by bertsky** [comment]
> Go ahead. Pull first though because I misread the output and thought the build had simply timed out and increased `no_output_timeout` and pushed to master, sorry about that.

Saw that. np. done :crossed_fingers:

**Comment by kba** [comment]
https://github.com/OCR-D/ocrd_all/commit/0a572239468b2e048e3fc3365347b9e5391d6387 fixed the build, an in a breezy 2 hours :)

**Pull request merged**